pkgs/tools/misc/ntfy-sh/default.nix

```diff
@@ -15,8 +21,19 @@ buildGoModule rec {
 
   doCheck = false;
 
+  nativeBuildInputs = [
+    debianutils
+    mkdocs
+    nodejs
+    python3
+    python3Packages.mkdocs-material
+    python3Packages.mkdocs-minify
+  ];
+
   preBuild = ''
-    make cli-deps-static-sites
+    sed -i 's# /bin/echo# echo#' Makefile
```

Comment:
Is there a reason this is a space, then `/bin/echo`? It looks like removing that would work fine, given [the context](https://github.com/binwiederhier/ntfy/blob/e0ad926ce9753a0ee5186a90a749d2333a9468ef/Makefile#L100).

(If it's better this way for whatever reason, keep it by all means -- I'm just curious :)

Reply:
I do this often in such scripts, to make sure it doesn't match an entry like /usr/bin/ or some other future addition. I agree it's not pretty and can be removed if preferred.

Reply:
I don't really have a preference.

An even better solution would be to remove this upstream -- there's no benefit to hardcoding `/bin/echo` over using it from PATH in that context. Feel free to PR yourself, or if you don't want to, I'm happy to :)